presto-kudu/src/test/java/com/facebook/presto/kudu/TestKuduIntegrationSmoke.java

```diff
@@ -70,6 +73,32 @@ public void testDescribeTable()
         assertEquals(filteredActual, expectedColumns, String.format("%s != %s", filteredActual, expectedColumns));
     }
 
+    @Test
+    public void testShowCreateTable()
```

Comment:
can you please another test when you try to set properties `table_id` and `num_replicas`

Reply:
Test has been extended to explicitly set `num_replicas` on creating the test table. `table_id`  has been removed.